kolibri/plugins/facility/assets/src/views/ManageClassPage/SelectableList.vue

```diff
@@ -77,7 +77,22 @@
           presentational
           :checked="isOptionSelected(option)"
           :label="option.label"
-        />
+        >
+          <template v-if="displayUserRole">
```

Comment:
(Blocking) We shouldn't be hard-coding this in the SelectableList component. The SelectableList component is meant to be a generic component that renders and accessible selectable searchable list. It shouldn't expose a `displayUserRole` prop and hardcoded coach role tag here.

What we can do instead to have more flexibility of how we render the options is to expose an `optionLabel` slot that takes the option as parameter. Such as:
```Vue
<slot name="optionLabel" :option="option"></slot>
```

and then in the parent component we can compute the label we have with the userTypeDisplay and all the styles we need,without making this general-purpose component specific to a given context.